app/main/views/frameworks.py

```diff
@@ -92,38 +85,54 @@ def framework_supplier_declaration():
     content = declaration_content.get_builder()
     status_code = 200
 
+    try:
+        page = int(request.args.get('page', 1))
```

Comment:
We used section ids for URLs in service edit, any reason why we can't do that here? We have the find section/next section logic for it and the URLs would make more sense than the pagination, since we're not really dealing with a list of items here.